src/_tests_/CountryNew.test.js

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react'
+import { BrowserRouter } from 'react-router-dom'
+import CountryNew from '../pages/CountryNew'
+
+describe('<CountryNew />', () => {
+    
+    beforeEach(() => {
+        render(
+            <BrowserRouter>
+                <CountryNew />
+            </BrowserRouter>
+        )
+    })
+
+  it("renders without crashing", () => {
+    render(
+      <BrowserRouter>
+        <CountryNew />
+      </BrowserRouter>
+    );
```

Comment:
Since you have the render on beforeEach() then it is not required on the test case